syndeo-template/src/main/java/com/google/cloud/syndeo/GenerateConfiguration.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo;
+
+import com.google.cloud.datapipelines.v1.BatchGetTransformDescriptionsResponse;
+import com.google.cloud.datapipelines.v1.MapType;
+import com.google.cloud.datapipelines.v1.TransformDescription;
+import com.google.cloud.datapipelines.v1.TypeName;
+import com.google.cloud.syndeo.common.ProviderUtil;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.schemas.transforms.SchemaTransformProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GenerateConfiguration {
+  private static final Logger LOG = LoggerFactory.getLogger(GenerateConfiguration.class);
+
+  public static void main(String[] args) throws IOException {
+    Collection<SchemaTransformProvider> providers = ProviderUtil.getProviders();
+    List<TransformDescription> descriptions =
+        providers.stream()
+            .filter(provider -> SyndeoTemplate.SUPPORTED_URNS.containsKey(provider.identifier()))
+            .map(GenerateConfiguration::providerToConfiguration)
+            .collect(Collectors.toList());
+
+    Files.write(
+        Paths.get("transform_configs.prototext"),
```

Comment:
Also missed this: I think the intent is that each config gets its own file. That said, I also don't see a way to set the directory where this file is written, so is the intent that this will just write the file, and then copy it over later, potentially splitting it up into separate files then?

I just want to make sure we're on the same page here.